submission_rules.adoc

```diff
@@ -267,6 +271,22 @@ A submission must contain the following:
 *   Result logs for each system-implementation tested
 
 
+The submission must be submitted by the submission deadline either as a pull request to the submission repository or using the <<id_submission_privacy,private submission process>>. 
+
+#### Qualified Exceptions [[id_qualified_exceptions]]
+
+In the case where last-minute unforeseen events prevent a submission from being submitted to the submission repository or sent to the results chair, a Late Submitter is allowed to make a submission after the deadline following the procedure outlined below. This procedure is meant for extreme circumstances beyond the submitter's control. No submitter may make a Late Submission in two consecutive rounds unless by unanimous approval from the other submitters.
+
+
+
+*   Prior to the submission deadline, the Late Submitter must provide a hash of a tarball containing the submission as Proof of Work as well as an explanation of the reason for the delay via email to the other submitters as well as the results chair, so that the other parties can anticipate a late submission. The submission tarball must contain <<id_submission_content, all required content>>. 
+*   Once notified of a late submission, the chair will wait until all late submissions are resolved before releasing any of the <<id_submission_privacy,private submission results>>.
+*   The submitter then has 24 hours after the submission deadline to make the submission tarball available to the results chair and other submitters. The tarball must match the hash. If the hash has been verified and the majority of other submitters do not object to the provided explanation, the results chair may accept the submission and make it and all other submissions available.  Otherwise the review committee must decide whether the Late Submission meets the criteria for acceptance, following the procedure described <<id_agenda_and_decisions,here>>.
+*   Failure to comply with the late submission procedure outlined here will result in the submission being deemed invalid except by unanimous approval from other submitters based on compelling explanation of extenuating circumstances. Examples of such cases include:
```

Comment:
It is easy to think of situations where "unanimous approval" will yield worse outcomes. This PR's intention seems aimed more at late submissions that were the submitter's fault (i.e., completely within the submitter's control), yet the "unanimous approval" stipulation will cause problems in both situations where the late submission **_was not_** the submitter's fault, and situations where it **_was_** the submitter's fault.

1. Situations where the late submission **_was not_** the submitter's fault: While it's true that the Austin snowpocalypse is an example received unanimous consent for late submissions, that does not mean that future natural disasters (or other major situations out of submitters' control) will also receive unanimous consent. As written, this PR opens the door for any single submitter to invalidate any number of other submitters' submissions if late due to disasters of this nature. I don't think anyone wants this to happen.
2. Situations where the late submission **_was_** the submitter's fault: As written, this PR closes the door for first-time submitters any amount of leeway with submission problems that cause a late submission. On prior WG calls, it seems that everyone agrees that this sort of leeway is reasonable and good to have. As @psyhtest mentioned, it is unlikely that unanimous approval will ever be achieved in this sort of situation. It seems best to some percentage majority to be the threshold, rather than unanimity. Also as written, this PR will statistically favor companies that push more resources toward their MLPerf effort (e.g., this will tend to be large companies), who will subsequently be more likely to get it Right The First Time<sup>TM</sup>.

My preference would be to preserve the power of the group of submitters to govern in these situations.

On a tangential and more general note, the more complicated the rules become, the less likely it is that a late submission will be able to follow the guidelines, even when that late submission normally would have been deemed acceptable. And this is where the ability of the group of submitters to govern becomes the most valuable.

Reply:
Per discussion in the working group, 24 hours will be extended to 72.

Our position on rules generally is that any submitter should be able to request that the rules be enforced as written. This protects small companies that may lack the implicit relationships that exists within the review committee.

However, it's worth noting that the review committee can change its schedule by 2/3 majority vote. I would expect that to happen in the case of a natural disaster (and it's what previously happened.) If the committee wants to change the schedule to account for a single submitter's lateness, it's free to do so, but that should take into consideration that submitters using the encrypted tarball process shouldn't need to reveal their data until the set of folks with repo access is locked down, and that there needs to be adequate time for review.

Reply:
Unanimous approval only comes into effect after 24 hours. After discussion in the inference WG today, there seemed to be a strong preference to extend this to 72 hours. We will amend the PR.

> As written, this PR closes the door for first-time submitters any amount of leeway with submission problems that cause a late submission.

What I remember about first-time submitters in the past is that they are typically able to submit on time, just not necessarily passing the submission checker. In that case, these rules will not come into effect.

> On a tangential and more general note, the more complicated the rules become

The issue is that right now, there are no rules on late submissions, which will lead to contention due to ambiguity.

Reply:
Understood, thanks for the clarification and for increasing the 24 hour limit to 72 hours.